### PR TITLE
NodeMaterial: PREM Update

### PR DESCRIPTION
**Pull request by sunag**
Issue: https://github.com/mrdoob/three.js/pull/18004

Unsolved issues:
- Fresnel reflection more hard in `NodeMaterial`.
Open example click in `animate` and after click in `nodes`.
https://raw.githack.com/sunag/three.js/dev-nodematerial-prem/examples/webgl_materials_envmaps_hdr_nodes.html

### CODE REVIEW

**Comment by Mugen87** [comment]
Merging for integration testing on `dev`.

File: examples/jsm/nodes/misc/TextureCubeUVNode.js [hunk 1]
```diff
@@ -165,17 +167,92 @@ TextureCubeUVNode.prototype = Object.create( TempNode.prototype );
...
+	
+	this.colorSpaceBL = this.colorSpaceBL || new ColorSpaceNode( new ExpressionNode( '', 'v4' ) );
+	this.colorSpaceBL.fromDecoding( builder.getTextureEncodingFromMap( this.value.value ) );
+	this.colorSpaceBL.input.parse( bilinearCubeUV.build( builder ) + '.bl' );
+	
+	this.colorSpaceBR = this.colorSpaceBR || new ColorSpaceNode( new ExpressionNode( '', 'v4' ) );
+	this.colorSpaceBR.fromDecoding( builder.getTextureEncodingFromMap( this.value.value ) );
+	this.colorSpaceBR.input.parse( bilinearCubeUV.build( builder ) + '.br' );
+	
+	var f = bilinearCubeUV.build( builder ) + '.f';
```
Comment (Mugen87) [inline examples/jsm/nodes/misc/TextureCubeUVNode.js]: It seems variable `f` is not used.

Comment (sunag) [reply examples/jsm/nodes/misc/TextureCubeUVNode.js]: Yes, is more simple remove without create other PR?

Comment (Mugen87) [reply examples/jsm/nodes/misc/TextureCubeUVNode.js]: see b78e8c431c7f90c28b14a580f073f675bcb30842 :+1:

Comment (sunag) [reply examples/jsm/nodes/misc/TextureCubeUVNode.js]: Thanks!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**